climada/entity/exposures/base.py

```diff
@@ -368,7 +368,8 @@ def get_impf_column(self, haz_type=''):
     def assign_centroids(self, hazard, distance='euclidean',
                          threshold=u_coord.NEAREST_NEIGHBOR_THRESHOLD,
                          overwrite=True):
-        """Assign for each exposure coordinate closest hazard coordinate.
+        """Wrapper for u_coord.assign_haz_centroids():
+        Assign for each exposure coordinate closest hazard coordinate.
```

Comment:
I shortened it a bit (especially in the imp.assign_centroids method). 
Since the parameter docstrings are often useful when using the function (as the editor directly shows you the documentation) I thought it may be useful to have them also in the wrapper methods. 
But of course it could also be further shortened